CHANGELOG.md

```diff
@@ -27,6 +28,7 @@
 * Update Kaniko executor to 1.6.0
 * Add support for separate control plane listener (disabled by default, available via the `ControlPlaneListener` feature gate)
 * Support for Dual Stack networking
+* Add support for filtering logs by the markers
```

Comment:
This should be removed I guess?

Reply:
No, the support of filtering by the markers is still a thing.

Reply:
But not in 0.23.